src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -221,6 +231,15 @@ public boolean isDependentArgumentAllowed(final Class<?> dependentClass) {
      */
     @Override
     public void validateArguments() {
+        // throw if any filter does not exists
+        final Set<String> nonExistant = new HashSet<>(disableFilters);
+        nonExistant.removeAll(readFilterNames);
+        if (!nonExistant.isEmpty()) {
+            throw new CommandLineException.BadArgumentValue(
+                    String.format("The read filter(s) specified do not exists: ",
+                            Utils.join(", ", nonExistant)));
+        }
```

Comment:
I'm assuming this is intended to be the fix for https://github.com/broadinstitute/gatk/issues/2397). I think it would be much cleaner and simpler to just integrate this check with the existing code below that already warns if a disabled filter is not enabled by the tool (I think you just need to add an if stmt there). Also, the error message (and comment, if you retain it) should specifically mention that its referring to a disabled filter.